schedule/yast/yast2_gui/yast2_gui_sle.yaml

```diff
@@ -10,6 +10,7 @@ vars:
   SOFTFAIL_BSC1063638: 1
   VALIDATE_ETC_HOSTS: 1
   YUI_REST_API: 1
+  SCC_ADDONS: sdk
```

Comment:
just noticed we are adding this setting to use the legacy module that install sle development module, which is pretty spaghetti code :) 
we are not using right now test data 'product' below. I believe @sofiasyria create solution to make this in simple way in 'scc_cleanup_reregister'.
But we can just take note of this for further work, at least to remove 'product' key would do.

Reply:
You're right. I've removed the `products` from test data. Thanks for the note.

Reply:
I will file a ticket as for what we agreed to have test suite which creates image with all packages being installed for libyui REST API.